code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm

```diff
@@ -309,8 +309,8 @@
 			target.show_laws()
 	//upgrade borgs
 	to_chat(user, "All unhacked AI's have been slaved to you. Now upgrading slaved borgs...")
-	command_announcement.Announce("There has recently been a security breach in the network firewall, the intruder has been shut out but we are unable to trace who did it or what they did.", "Network Monitoring")
-	sleep(600) //1 minute delay for balance purposes

```

Comment:
Keep the synthetic takeover times as they are.
They only have a minor effect on the overall time but serve to avoid a quick takeover with almost no warning.

Reply:
The takeover doesn't do much on its own. It revolves around the robot transformer. It's placement isn't a "round ender" like the other trees are and my original plan was to see how powerful it was before changing the time accordingly. The transformer can be and has been quite underwhelming largely depending on the competency of the cyborgs, so it's only fair for the AI to get access to it sooner unless major adjustments are made to the usefulness of it.